arctic-report-card/ARCRC.ttl

```diff
@@ -232,508 +262,647 @@ Find current and past assessments of the Arctic Report Card Vital Signs by visit
+odo:ARCRC_00000015 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000512 ;
+                   obo:IAO_0000115 "Subsea permafrost in the Arctic is generally relict terrestrial permafrost, inundated after the last glaciation and now degrading under the overlying shelf sea." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Subsea Permafrost Indicator"@en ;
+                   skos:broadMatch "http://www.purl.dataone.org/odo/ECSO_00010130" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000016
+odo:ARCRC_00000016 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Applying and centering Indigenous perspectives and observations of Arctic change" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Working with Communities to Observe the Arctic"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000017
+odo:ARCRC_00000017 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Here, we refer to “western Alaska” salmon as those spawning in watersheds that drain into the eastern Bering Sea." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Western Alaska Salmon"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/SALMON_00000215" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000018
+odo:ARCRC_00000018 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Any product of the condensation of atmospheric water vapor that falls under gravity" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Precipitation.'" ;
+                   rdfs:label "precipitation"@en ;
+                   owl:sameAs "http://purl.obolibrary.org/obo/ENVO_01000703" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000019
+odo:ARCRC_00000019 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Estimated total amount of ice in the sediment column and represents total ice content" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Subsea Permafrost'" ;
+                   rdfs:label "total ice content"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/ECSO_00002173" .
```

Comment:
is this more than a relatedMatch? Is it a synonym? Best to use the srongest relationship that always holds true. Let's discuss.

Reply:
In the[essay](https://arctic.noaa.gov/report-card/report-card-2023/permafrost-beneath-arctic-ocean-margins-2023/), it says total ice content is in cubic meters per square meter, so I think total ice content can be a narrower match to volumetric ice content.

Reply:
Changed it to a narrowMatch
`skos:narrowMatch "http://purl.dataone.org/odo/ECSO_00002173"`